parsons/bluelink/bluelink.py

```diff
@@ -0,0 +1,86 @@
+from parsons.utilities.api_connector import APIConnector
+from parsons.utilities import check_env
+from parsons.bluelink.person import Person
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+
+API_URL = 'https://api.bluelink.org/webhooks/'
+
+
+class Bluelink:
+    """
+    Instantiate a Bluelink connector
+    Allows for a simple method of inserting Person data to Bluelink via a webhook
+
+    `Args:`:
+        user: str
+            Bluelink webhook user name
+        password: str
+            Bluelink webhook password
+    """
+    def __init__(self, user=None, password=None):
+        self.user = check_env.check('BLUELINK_WEBHOOK_USER', user)
+        self.password = check_env.check('BLUELINK_WEBHOOK_PASSWORD', password)
+        print(self.user)
+        print(self.password)
+        self.headers = {
+            "Content-Type": "application/json",
+        }
+        self.api_url = API_URL
+        self.api = APIConnector(self.api_url, auth=(self.user, self.password), headers=self.headers)
+
+    def upsert_person(self, source, person=None):
+        """
+        Upsert A person object into Bluelink.
+        For an update:
```

Comment:
Should this say "For an upsert" to be consistent?

Reply:
I'll try to make it more clear that I'm trying to indicate what the conditions for an update instead of an insert.